tests/unit/test_ml_utils.py

```diff
@@ -31,14 +31,7 @@ def test_ml_class_mappings(gpu):
         lightgbm = None
         xgboost = None
 
-    if gpu:
-        classes_dict = get_gpu_classes()
-    else:
-        # Imports needed to use sklearn.experimental classes
-        from sklearn.experimental import enable_halving_search_cv  # noqa: F401
-        from sklearn.experimental import enable_iterative_imputer  # noqa: F401
```

Comment:
This was left over from when I was literally listing every single class in the `get_cpu_classes` DataFrame, but since we use sklearn's `all_estimators` function these imports are not needed anymore.